enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -82,7 +82,7 @@ func (i *InvocationStatService) getTrendBasicQuery(timezoneOffsetMinutes int32)
 	q := ""
 	if i.isOLAPDBEnabled() {
 		q = fmt.Sprintf("SELECT %s as name,", i.olapdbh.DateFromUsecTimestamp("updated_at_usec", timezoneOffsetMinutes)) + `
-	    SUM(duration_usec) as total_build_time_usec,`
+	    SUM(CASE WHEN duration_usec > 0 THEN duration_usec END) as total_build_time_usec,`
```

Comment:
since this now become the same whether clickhouse is enabled or not, we can move it below?

Reply:
yeah yes done